Src/Newtonsoft.Json.UnityConverters.Tests/ConvertingUnityTypes/VectorTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿using UnityEngine;
+
+namespace Newtonsoft.Json.UnityConverters.Tests.ConvertingUnityTypes
+{
+    public class Vector2Tests : TypeTester<Vector2>
+    {
+        public static (Vector2 deserialized, string serialized)[] representations = new[] {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=4930242)